neuraxle/steps/column_transformer.py

```diff
@@ -0,0 +1,110 @@
+from typing import List, Tuple, Union
+
+import numpy as np
+
+from neuraxle.base import BaseStep, NonFittableMixin, MetaStepMixin
+from neuraxle.pipeline import Pipeline
+from neuraxle.steps.loop import ForEachDataInput
+from neuraxle.union import FeatureUnion
+
+ColumnSelectionType = Union[Tuple[int, BaseStep], Tuple[List[int], BaseStep], Tuple[slice, BaseStep]]
+ColumnChooserTupleList = List[ColumnSelectionType]
+
+
+class ColumnSelector2D(NonFittableMixin, BaseStep):
+    """
+    A ColumnSelector2D selects column in a sequence.
+    """
+
+    def __init__(self, columns_selection: ColumnSelectionType):
+        super().__init__()
+        self.column_selection = columns_selection
+
+    def transform(self, data_inputs):
+        if isinstance(self.column_selection, int):
```

Comment:
Would be cool to fix #185 by adding an if there that converts a range to a slice.